app/views/services/show.html.erb

```diff
@@ -7,17 +7,19 @@
 </div>
 
 <div class="row ctaRow service-show">
-  <% unless @related_services.blank? %>
+  <% unless @services.blank? %>
```

Comment:
These changes seem unrelated to Finding Aid?

Reply:
See comment above re: branch cleanup